test/config/utility.cc

```diff
@@ -116,6 +116,38 @@ const std::string ConfigHelper::HTTP_PROXY_CONFIG = BASE_CONFIG + R"EOF(
             name: route_config_0
 )EOF";
 
+const std::string ConfigHelper::QUIC_HTTP_PROXY_CONFIG = BASE_UDP_LISTENER_CONFIG + R"EOF(
```

Comment:
I wonder if this should be a config helper vs. a full new config? I will defer to @alyssawilk on this one.

Reply:
I'd mildly prefer this to be a standard config modifier / helper, especially as eventually I'd like to run the H1/H2 tests with H1/H2/H3.  Eventually I think we'll want to just apply the quic config modifiers when the client / upstream type is H3 and I think that'll be easier if we use standard config, so we don't get out of sync if someone tweaks the standard access loggers or some such.

Reply:
Okay, added a TODO to generalize an HttpProxyConfig for both TCP and QUIC